project_task_code/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Sequential Code for Tasks",
-    "version": "15.0.1.0.3",
+    "version": "15.0.1.0.4",
```

Comment:
The bot will auto-bump the version

So, it is not even needed here

Reply:
Done!